pkg/features/features.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+import (
+	"k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+	VolumeInUseErrorHandler featuregate.Feature = "VolumeInUseErrorHandler"
```

Comment:
traditionally feature-gates are typically verbs in Kubernetes. Such as - "HandleVolumeInUseError" . Also now that I think more about this - I don't like this to be a feature-gate because a feature-gate is designed to go away eventually but it looks like this is something we will keep forever and hence does not belong in a feature-gate. 

I am sorry for flip flopping on this. :(

Reply:
cc @msau42 ^

Reply:
I am fine with making it a separate flag. I initially thought a feature gate may be useful to reduce the number of flags we expose, but now that I think about it more, a feature gate is less discoverable than a flag.

Reply:
Ack. Will create a flag. Should I still keep the feature gate changes, given that @gnufied  wanted this for volume expansion cancellation feat?

Reply:
nah its okay to keep this PR to limited to what we need, I can bring back the feature-gate thingy in my PR.

Reply:
Ack